jetty-http/src/main/java/org/eclipse/jetty/http/compression/HuffmanDecoder.java

```diff
@@ -11,20 +11,30 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.http3.qpack.internal.util;
+package org.eclipse.jetty.http.compression;
 
 import java.nio.ByteBuffer;
 
-import org.eclipse.jetty.util.Utf8StringBuilder;
+import org.eclipse.jetty.util.CharsetStringBuilder;
+
+import static org.eclipse.jetty.http.compression.Huffman.rowbits;
+import static org.eclipse.jetty.http.compression.Huffman.rowsym;
 
 public class HuffmanDecoder
 {
-    static final char EOS = HuffmanEncoder.EOS;
-    static final char[] tree = HuffmanEncoder.tree;
-    static final char[] rowsym = HuffmanEncoder.rowsym;
-    static final byte[] rowbits = HuffmanEncoder.rowbits;
+    public static String decode(ByteBuffer buffer, int length) throws EncodingException
```

Comment:
Didn't we agree to get rid of this method, as it encourages inefficient usage of temporary decoder?

Reply:
Yep, I removed the usage but didn't remove this.
Done.